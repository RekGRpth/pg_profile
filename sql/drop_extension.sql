/* Testing drop server with data */
SELECT * FROM profile.drop_server('local');
DROP EXTENSION pg_profile;
DROP EXTENSION IF EXISTS pg_stat_statements;
DROP EXTENSION IF EXISTS dblink;
DROP SCHEMA profile;
DROP SCHEMA dblink;
DROP SCHEMA statements;
