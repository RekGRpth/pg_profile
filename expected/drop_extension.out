/* Testing drop server with data */
SELECT * FROM profile.drop_server('local');
 drop_server 
-------------
           1
(1 row)

DROP EXTENSION pg_profile;
DROP EXTENSION IF EXISTS pg_stat_statements;
DROP EXTENSION IF EXISTS dblink;
DROP SCHEMA profile;
DROP SCHEMA dblink;
DROP SCHEMA statements;
